projects_test.go

```diff
@@ -192,3 +192,75 @@ func ExampleProjectsService_ListProjects() {
 	// Output:
 	// infra/infra/infra_libs - State: ACTIVE
 }
+
+func TestProjectsService_GetBranch(t *testing.T) {
+	setup()
+	defer teardown()
+
+	existBranches := map[string]*gerrit.BranchInfo{
+		"branch": &gerrit.BranchInfo{
+			Ref:       "123",
+			Revision:  "abcd1234",
+			CanDelete: true,
+		},
+		"branch/foo": &gerrit.BranchInfo{
+			Ref:       "456",
+			Revision:  "deadbeef",
+			CanDelete: false,
+		},
+	}
+
+	testMux.HandleFunc("/projects/go/branches/", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+
+		branchName := r.URL.Path[len("/projects/go/branches/"):]
```

Comment:
(note that the branchName is already transformed back to unescaped slashes so it didn't actually provide actual value to what's been changed in the current PR :crying_cat_face: )

Reply:
Seems to be a Gerrit vs. Test behavior here. Thanks anyway!